src/dsp_tools/commands/validate_data/make_data_rdf.py

```diff
@@ -197,3 +202,10 @@ def _transform_uri_value(val: ValueDeserialised, res_iri: URIRef) -> ValueRDF:
         else Literal("", datatype=XSD.string)
     )
     return UriValueRDF(URIRef(val.prop_name), content, res_iri)
+
+
+def _transform_file_value(val: AbstractFileValueDeserialised) -> AbstractFileValueRDF:
```

Comment:
This function will be extended through the coming PRs.
Currently we only check the cardinalities, later on we identify which file type is given through the extension.
This will be implemented separately. I will likely keep `GenericFileValueRDF` for extensions we do not know.